src/index.html

```diff
@@ -2,12 +2,55 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars stars--0">
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+      <li class="stars__star"></li>
+    </ul>
+    <ul class="stars stars--1">
```

Comment:
```suggestion
    <ul class="stars stars--0">
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
    </ul>
    
    <ul class="stars stars--1">
```

Add blank line between multiple sibling blocks of html